### PR TITLE
Custom headers documentation

### PR DESCRIPTION
**Pull request by Robinfr**
Added a bit of documentation on the custom headers configuration. Please note, there will be more in-depth documentation changes in the future.

### CODE REVIEW

**Comment by ConnorLand** [comment]
Sounds great. Thanks for the perfect PR @Robinfr .

File: content/en/docs/refguide/modeling/app-explorer/app/app-settings/configuration.md [hunk 1]
```diff
@@ -70,13 +71,30 @@ Private configuration values are available in Studio Pro 10.9.0 and above. In ve
...
 
 Each custom setting consists of a name and a value. For example, to set the hash algorithm to BCRYPT you add a custom setting with name `HashAlgorithm` and value `BCRYPT`.
 
 For settings overview, see [Runtime Customization](/refguide/custom-settings/).
 
+### Headers {#headers}
+
+You can specify custom headers that will be applied to responses coming from the Mendix Runtime. 
+
+{{% alert color="warning" %}}
```
Comment (ConnorLand) [inline content/en/docs/refguide/modeling/app-explorer/app/app-settings/configuration.md:85]: @Robinfr is there any guide/teaching resource we can link people to in this alert? We want to try and enable people to use the functionality, while also cautioning them.

File: content/en/docs/refguide/modeling/app-explorer/app/app-settings/configuration.md [hunk 2]
```diff
@@ -70,13 +71,30 @@ Private configuration values are available in Studio Pro 10.9.0 and above. In ve
 You can use custom server settings to configure the Runtime beyond the standard possibilities offered by Studio Pro.
 
 {{% alert color="warning" %}}
```
Comment (ConnorLand) [inline content/en/docs/refguide/modeling/app-explorer/app/app-settings/configuration.md:73]: @Robinfr is there any guide/teaching resource we can link people to in this alert? We want to try and enable people to use the functionality, while also cautioning them.

File: content/en/docs/refguide/modeling/app-explorer/app/app-settings/configuration.md (hunk 1 above)
Comment (Robinfr) [reply content/en/docs/refguide/modeling/app-explorer/app/app-settings/configuration.md:85]: Hmm not really at the moment. We've added a list of "forbidden" custom headers in Studio Pro that developers cannot set, so the worst case scenarios are covered. My fear is that there might be some headers that we do allow, but would override a header that's being set from the runtime, that could cause weird behaviour.

File: content/en/docs/refguide/modeling/app-explorer/app/app-settings/configuration.md (hunk 2 above)
Comment (Robinfr) [reply content/en/docs/refguide/modeling/app-explorer/app/app-settings/configuration.md:73]: @ConnorLand This is not new :) But same goes for this one, we don't have specific guides for this. In this case, you can check the custom server settings list to see what the potential impact could be of setting something.

**Review by quinntracy — APPROVED** [review]

**Pull request merged**